tests/driver_stm32_eth/Makefile

```diff
@@ -0,0 +1,19 @@
+BOARD ?= nucleo-f207zg
+
+include ../Makefile.tests_common
+
+USEMODULE += event_thread
+USEMODULE += l2util
+USEMODULE += od
+USEMODULE += ps
+USEMODULE += shell
+USEMODULE += shell_commands
+
+# the driver to test
+USEMODULE += stm32_eth
+
+FEATURES_REQUIRED += periph_eth
+# TODO: complete with other STM32 CPUs that have ethernet
+FEATURES_REQUIRED_ANY += cpu_stm32f2|cpu_stm32f7
```

Comment:
Note that `FEATURES_REQUIRED += periph_eth` should only select STM32 MCUs that do have ethernet. So no need to filter out MCU families.

Reply:
But then how to limit to stm32? ESP also has `periph_eth`.

Reply:
Ah, OK!

How about `FEATURES_REQUIRED += cpu_stm32` then?

Reply:
True, much direct. I'll change!

Reply:
On a second thought, the `cpu_stm32` feature should be required by the `stm32_eth` driver itself, so I'll add it there.